ibis/util.py

```diff
@@ -530,7 +530,7 @@ def _removeprefix(text, prefix):
         source = _removeprefix(source, f"{prefix}://")
 
     def _absolufy_paths(name):
-        if not name.startswith(("http", "s3")):
+        if not name.startswith(("http", "s3", "az", "abfs", "abfss", "adl", "gs")):
```

Comment:
should `https` also be in here?

Reply:
`http` will cover that case

Reply:
We'll need to expand this list elsewhere, but I'll do that in a follow-up.